src/guide/contributing/translations.md

```diff
@@ -1,11 +1,11 @@
-# Translations
+# Traducciones
 
-Vue has spread across the globe, with the core team being in at least half a dozen different timezones. [The forum](https://forum.vuejs.org/) includes 7 languages and counting and many of our docs have [actively-maintained translations](https://github.com/vuejs?utf8=%E2%9C%93&q=vuejs.org). We'are very proud of Vue's international reach, but we can do even better.
+Vue se ha expandido, con el equipo del núcleo estando en, al menos, mitad de una docena de zonas de tiempo diferentes. [El foro](https://forum.vuejs.org/) incluye 7 lenguajes y en expansión y muchos de nuestros documentos poseen [traducciones mantenidas activamente](https://github.com/vuejs?utf8=%E2%9C%93&q=vuejs.org). Estamos muy orgullosos de que Vue se expandió internacionalmente, pero podemos hacerlo mejor.
```

Comment:
I would like to suggest, keep the translation of this paragraph like this: https://es.vuejs.org/v2/guide/join.html#Traducir-Documentos

Reply:
Updated!